server/market/routers_test.go

```diff
@@ -412,6 +414,7 @@ func (u *tUTXO) Value() uint64                   { return u.val }
 func (u *tUTXO) FeeRate() uint64                 { return 0 }
 func (u *tUTXO) RedeemScript() []byte            { return nil }
 func (u *tUTXO) LockTime() time.Time             { return time.Time{} }
+func (u *tUTXO) TxData() []byte                  { return nil }
```

Comment:
Would you mind nuking the old Contract interface method subs?  (SwapAddress, RedeemScript, LockTime)

And actually I think the TxData method is legacy too.